scss/bootstrap/overrides/style/_type.scss

```diff
@@ -27,8 +27,13 @@ h6, .h6 { font-weight: $crm-font-weight-h6; }
   margin: 0 0 15px 0;
 
   > h1,
-  > h2 {
+  > h2,
+  > h3,
+  > h4,
+  > h5,
+  > h6 {
     font-size: em(18);
+    font-weight: $crm-font-weight-h1;
```

Comment:
Why is it always `h1`, as I can see from the variables, we have
```
$crm-font-weight-h1:          600 !default;
$crm-font-weight-h2:          600 !default;
$crm-font-weight-h3:          600 !default;
$crm-font-weight-h4:          500 !default;
$crm-font-weight-h5:          700 !default;
$crm-font-weight-h6:          500 !default;
```
So it seems the font-weight is supposed to be different for individual `h{n}`tags. At this point I am not even sure if the variables are necessary. Maybe a single font-weight value for all `h{n}` is enough? In that case the variables should be cleaned up to have only a single variable. 

Also we have a variable `$headings-font-weight`. Not sure how it is different from `$crm-font-weight-h{n}`. Could you see if all of them are necessary? If not then we should have a single variable.